nwbinspector/utils.py

```diff
@@ -57,6 +57,15 @@ def is_ascending_series(series: np.ndarray, nelems=None):
     return np.all(np.diff(series[:nelems]) > 0)
 
 
+def get_uniform_indexes(length: int, nelems: Optional[int] = 200):
+    """General purpose function for generating nelems indices from 0 to length-1 with approximately equal gaps."""
+    indexes = np.round(np.linspace(start=0, stop=length - 1, num=nelems)).astype(int)
+    if nelems is None or nelems >= length - 1:
+        return slice(0, None)
+    else:
+        return indexes
```

Comment:
fancy indexing is going to be expensive. Could you use slice indexing instead?

Reply:
Done